tools/src/verdaccio/utils.ts

```diff
@@ -1,5 +1,9 @@
 import { execSync } from 'node:child_process';
-import { RegistryData } from './types';
+import { RegistryData } from './start-local-registry';
+
+export function uniquePort(): number {
+  return Number((6000 + Number(Math.random() * 1000)).toFixed(0));
+}
```

Comment:
Unused, `global-setup.e2e.ts` duplicates this expression.

Reply:
reused this fn now in `global-setup.e2e.ts`